deploy/compliance-operator-chart/templates/role.yaml

```diff
@@ -0,0 +1 @@
+../../role.yaml
```

Comment:
These two are still symlinks, but I don't think we generate these like we to the CRD files.

Helm will still balk at them, but we can either symlink or we can duplicate them.